lib/synsbasen_api/resources/version.rb

```diff
@@ -3,15 +3,15 @@
 module SynsbasenApi
   # The `Version` class provides methods for interacting with version-related
   # endpoints in the Synsbasen API.
-  class Version < Client
+  class Version < Resource
     class << self
       # Retrieves information about a specific version based on its ID.
       #
       # @param id [String] The unique identifier of the version.
       # @return [ApiResponse] An instance of `ApiResponse` containing details
       #   of the specified version.
       def find(id, expand: [])
-        get("/v1/versions/#{id}", expand: expand)
+        get("/v1/#{resource_name}/#{id}", expand: expand)
       end
```

Comment:
All the finds are identical at this point too. Perhaps we need a findable module too?

Reply:
Great idea! Implemented.